service/src/main/java/com/fecklessweasel/service/datatier/UniversityTable.java

```diff
@@ -23,6 +23,8 @@
     private static String INSERT_ROW = "insert into University (longName, acronym, city, state, country) values (?,?,?,?,?)";
     /** Lookup user query. */
     public static String LOOKUP_UNIVERSITY_QUERY =
+            "SELECT * FROM University WHERE University.longName=?";
```

Comment:
This should be removed. Looking up a university by it's long name is semantically incorrect because the university can be renamed or otherwise updated. Long names are also not guaranteed to be unique.